### PR TITLE
Configure alternative BigNumeric precision and scale defaults

### PR DESCRIPTION
**Pull request by isha97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -908,6 +908,26 @@ word-break:break-word
      </td>
      <td>Read</td>
```
Comment (davidrabinowitz) [inline README.md:909]: `README.md` should not be edited, just `README-template.md`

File: README.md [hunk 2]
```diff
@@ -908,6 +908,26 @@ word-break:break-word
...
    </tr>
+  <tr>
+     <td><code>bigNumericFieldsPrecision</code>
+     </td>
+     <td>A config to specify the precision to use while reading all the BigNumeric fields of a table.
```
Comment (davidrabinowitz) [inline README.md:914]: Should be `An alternative default precision for BigNumeric fields, as the BigQuery default is too wide for Spark. Values can be between 1 and 38.  This default is used only when the field has an unparameterized BigNumeric type.`

File: README.md [hunk 3]
```diff
@@ -908,6 +908,26 @@ word-break:break-word
...
+          Please note that there might be data loss if the actual data's precision is more than what is specified.
+          By default this is not set and precision from the field is used.
+         <br/> (Optional)
+     </td>
+     <td>Read</td>
+   </tr>
+  <tr>
+     <td><code>bigNumericFieldsScale</code>
+     </td>
+     <td>A config to specify the scale to use while reading all the BigNumeric fields of a table.
```
Comment (davidrabinowitz) [inline README.md:924]: Should be `An alternative default scale for BigNumeric fields. Values can be between 0 and 38, and less then bigNumericFieldsPrecision.  This default is used only when the field has an unparameterized BigNumeric type.`

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]
Please update the CHANGES.md file

File: README.md (hunk 1 above)
Comment (isha97) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (isha97) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (isha97) [reply README.md]: Fixed

File: spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java [hunk 4]
```diff
@@ -167,6 +168,9 @@ public static WriteMethod from(@Nullable String writeMethod) {
 
   public static final String GPN_ATTRIBUTION = "GPN";
 
+  public static final String BIG_NUMERIC_DEFAULT_PRECISION = "bigNumericDefaultPrecision";
```
Comment (davidrabinowitz) [inline spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java:171]: The parameter names should match the documentation (and exceptions)

**Review by davidrabinowitz — APPROVED** [review]

File: spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java (hunk 4 above)
Comment (isha97) [reply spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java:171]: Done

**Pull request merged**